include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -76,6 +76,8 @@ ERROR(error_unknown_arg,none,
       "unknown argument: '%0'", (StringRef))
 ERROR(error_invalid_arg_value,none,
       "invalid value '%1' in '%0'", (StringRef, StringRef))
+ERROR(error_invalid_locale_code,none,
+      "unsupported locale code; supported locale codes are '%0'", (StringRef))
```

Comment:
probably `are: ` instead since it's going to be a comma-separated list that follows.